gallery/templates/view_dir.html

```diff
@@ -35,7 +35,7 @@
 {% if children|length >= 1 %}
     <div class="row" id="child_list">
     {% for child_type, child in children %}
-        {% if not child.hidden or (auth_dict['is_eboard'] or auth_dict['is_rtp'])  or auth_dict['is_alumni'] %}
+        {% if not child.hidden or (auth_dict['is_eboard'] or auth_dict['is_rtp'] or auth_dict['is_alumni']) and not member_lock %}
```

Comment:
```suggestion
        {% if not child.hidden or (auth_dict['is_eboard'] or auth_dict['is_rtp'] or auth_dict['is_alumni']) and not session['userinfo'].get('member-lock') %}
```
does this work? can't remember if session will be defined in the context of jinja